io/disk/multipath_test.py

```diff
@@ -151,46 +151,175 @@ def test(self):
                 if disk in multipath.get_paths(path_dic["wwid"]):
                     msg += "Blacklist of %s fails\n" % disk
             multipath.form_conf_mpath_file(defaults_extra=plcy)
+        if msg:
+            self.fail("Some tests failed. Find details below:\n%s", msg)
+
+        self.test_fail_paths()
+
+    def test_fail_paths(self):
+        '''
+        calling all function from here
+        '''
+        self.test_fail_reinstate_individual_paths()
+        self.test_io_run_on_single_path()
+        self.test_failing_reinstate_all_paths()
+        self.test_removing_all_paths()
+        self.test_suspend_resume_individual_mpath()
+        self.test_suspend_resume_all_mpath()
```

Comment:
why calling one test from another ?

Reply:
when we run the complete test, we need these functions also to be run right?

Reply:
avocado (unittest) calls all `test_X()` by default. You need not call them.

Reply:
ohhh ok, then will remove the function.
please let me know if you any more comments. I want to complete it on one shot

Reply:
@Naresh-ibm This  changes the flow of the test. Once you address this only we can look at the code in a meaningful way and review. Hope you understand that.